src/applications/edu-benefits/1995/containers/ConfirmationPage.jsx

```diff
@@ -39,6 +39,7 @@ class ConfirmationPage extends React.Component {
       : {};
     const name = form.data.veteranFullName;
     const benefit = form.data.benefit;
+    const printPreviewColon = <div className="print-only colon">:</div>;
```

Comment:
Same with this; `vads-u-display--inline` could be used here too.

Reply:
I'll update it